db/common.go

```diff
@@ -0,0 +1,8 @@
+package db
+
+// ErrorResponse - a struct representing a response for an error
```

Comment:
struct name itself suggest its purpose here, so we can avoid putting comment
same follow at other places in code

Reply:
Removed unnecessary comments.